### PR TITLE
rename or remove overview sections

### PR DESCRIPTION
**Pull request by BeateRixen**
We had inconsistent usage of "overview" and "introduction" in section headers. I unified this towards "introduction" as this usually is what the content is about.
Moreover, in several cases, I removed this header entirely, having a first paragraph at the top without a subheading. This was the preferred layout the doc team agreed on.

@kailol-sag I did not touch the Streaming Analytics section. There is also at least one file name called "Overview" but I leave it up to you if you want to keep it.

### CODE REVIEW

**Comment by kailol-sag** [comment]
> @kailol-sag I did not touch the Streaming Analytics section. There is also at least one file name called "Overview" but I leave it up to you if you want to keep it.

@BeateRixen , please feel free to also rename https://softwareag-c8y-docs-build-pr-1857.surge.sh/streaming-analytics/overview-analytics/ to "Introduction". Guess you'll also have to change the file name then and adapt all links to that page ...

And we should also tell the dev team. Don't know if they have links from the UI ti that page which they would then have to change.

**Comment by BeateRixen** [comment]
> > @kailol-sag I did not touch the Streaming Analytics section. There is also at least one file name called "Overview" but I leave it up to you if you want to keep it.
> 
> @BeateRixen , please feel free to also rename https://softwareag-c8y-docs-build-pr-1857.surge.sh/streaming-analytics/overview-analytics/ to "Introduction". Guess you'll also have to change the file name then and adapt all links to that page ...
> 
> And we should also tell the dev team. Don't know if they have links from the UI to that page which they would then have to change.

@kailol-sag I will leave this for now. Since this may affect the software I leave it up to you and the dev team to adapt this.

**Comment by BeateRixen** [comment]
@pawel-rynarzewski-sag I unified a couple of files towards a consistent usage of "introduction" instead of "overview" and merged several files. Could you please check if this affects links from the software? I think most of them are not critical (the ones in the Concepts section, for example). But for those in device-integration or protocol-integration we might have links. Or should I better add redirects?

File: content/dtm/asset-hierarchy-bundle/introduction.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (BeateRixen) [inline content/dtm/asset-hierarchy-bundle/introduction.md:1]: @lra-sag For consisteny, I would like to rename this file. Can you please check if this link is used in the software or somewhere else? Should we better use a redirect?

**Comment by pawel-rynarzewski-sag** [comment]
> @pawel-rynarzewski-sag I unified a couple of files towards a consistent usage of "introduction" instead of "overview" and merged several files. Could you please check if this affects links from the software? I think most of them are not critical (the ones in the Concepts section, for example). But for those in device-integration or protocol-integration we might have links. Or should I better add redirects?

Found no links to `device-integration`. For `protocol-integration`, we only link to `/docs/protocol-integration/device-protocols`. It looks like we only need to update these: https://github.softwareag.com/IOTA/cumulocity-ui/pull/6051.

**Comment by BeateRixen** [comment]
> > @pawel-rynarzewski-sag I unified a couple of files towards a consistent usage of "introduction" instead of "overview" and merged several files. Could you please check if this affects links from the software? I think most of them are not critical (the ones in the Concepts section, for example). But for those in device-integration or protocol-integration we might have links. Or should I better add redirects?
> 
> Found no links to `device-integration`. For `protocol-integration`, we only link to `/docs/protocol-integration/device-protocols`. It looks like we only need to update these: https://github.softwareag.com/IOTA/cumulocity-ui/pull/6051.

Thanks @pawel-rynarzewski-sag I have added a redirect for this link but it might still be cleaner to change it.

**Comment by pawel-rynarzewski-sag** [comment]
> > > @pawel-rynarzewski-sag I unified a couple of files towards a consistent usage of "introduction" instead of "overview" and merged several files. Could you please check if this affects links from the software? I think most of them are not critical (the ones in the Concepts section, for example). But for those in device-integration or protocol-integration we might have links. Or should I better add redirects?
> > 
> > 
> > Found no links to `device-integration`. For `protocol-integration`, we only link to `/docs/protocol-integration/device-protocols`. It looks like we only need to update these: https://github.softwareag.com/IOTA/cumulocity-ui/pull/6051.
> 
> Thanks @pawel-rynarzewski-sag I have added a redirect for this link but it might still be cleaner to change it.

From UI apps we don't link to any specific protocol pages, just to the main one.

File: content/dtm/asset-hierarchy-bundle/introduction.md (hunk 1 above)
Comment (lra-sag) [reply content/dtm/asset-hierarchy-bundle/introduction.md:1]: @BeateRixen, we don't have any direct references and the file can be renamed.
@aik24, FYI

**Review by pawel-rynarzewski-sag — APPROVED** [review]
@BeateRixen, yes :+1:

**Pull request merged**